src/com/t_oster/visicut/model/graphicelements/svgsupport/SVGImporter.java

```diff
@@ -197,6 +200,33 @@ public double determineResolution(File f, List<String> warnings)
           {
             WwwInkscapeComSeen = true;
           }
+          if (line.contains("inkscape:version="))
+          {
+	    // inkscape:version="0.92.0 ...."
+	    // inkscape:version="0.91 r"
+
+	    // FIXME: version number comparison needed here!
+	    Pattern versionPattern = Pattern.compile("inkscape:version\\s*=\\s*[\"']?([0-9]+)\\.([0-9]+)");
+	    Matcher matcher = versionPattern.matcher(line);
+	    if (matcher.find())
+	      {
+	        String vers = matcher.group();
+		// vers = "inkscape:version=\"0.92"
+
+	        int vers_maj;
+		try { vers_maj = Integer.parseInt(matcher.group(1)); }
+		catch (NumberFormatException e) { vers_maj = -1; }
+
+	        int vers_min;
+		try { vers_min = Integer.parseInt(matcher.group(2)); }
+		catch (NumberFormatException e) { vers_min = -1; }
+
+		if ((vers_maj >= 0) && (vers_min >= 92))
```

Comment:
I would suggest checking for  `((vers_maj == 0) && (vers_min >= 92)) || (vers_maj > 0)` so that we also handle a future version "1.xx".